### ISSUE TITLE
Contribution guide for Quickstart

### ISSUE THREAD

**Issue by msfussell** [dapr/quickstarts#967]
This issue is to provide guidance on contributions to the Quickstart repo, including
- Mechanical Markdown
- How to create a sample based on an issue.

**Comment by msfussell** [comment]
@marcduiker for viz

**Comment by paulyuk** [comment]
I agree with this idea.  Starting now as I go through the process for 1.13 :)

**Comment by marcduiker** [comment]
> I agree with this idea. Starting now as I go through the process for 1.13 :)

Great! Once the written version is there, I'll create a video about contributing to the Quickstarts repo.
